source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm

```diff
@@ -303,7 +302,7 @@ void applyBackgroundImageConstraints(const BackgroundImage *backgroundImagePrope
                 isDeficientInHeight = YES;
             }
 
-            if (isDeficientInWidth and isDeficientInWidth) {
+            if (isDeficientInWidth and isDeficientInHeight) {
```

Comment:
>cientInWidth and isDeficientInHei [](start = 22, length = 33)

ha! good catch. How do we catch bugs like these sooner rather than later? Can we have a unit test added that will invoke this code path? Also do we have an end-end functional/scenario sample that tests this out? If not let's get it added/test results updated for the UWP app etc.